### ISSUE TITLE
Commandline parameters for filtering not working (for me:)

### ISSUE THREAD

**Issue by fbehrens** [haf/expecto#99]
Hi, and please bare with me if I miss something, but ... 

Can't get filtering working
```
$ mono src/eexpecto/bin/Release/eexpecto.exe --help
...
--filter-test-list <substring>                        filters the list of test lists by a substring.
--filter-test-case <substring>                          filters the list of test cases by a substring.
--run [<tests>...]    runs only provided tests.

$ mono src/eexpecto/bin/Release/eexpecto.exe --filter-test-list aa --summary                                                                     
...
Passed:  3
	tl1/tc1
	tl1/tc2
	tc3

$ mono src/eexpecto/bin/Release/eexpecto.exe --filter-test-case  aa --summary                                                                    
...
Passed:  3
	tl1/tc1
	tl1/tc2
	tc3

$ mono src/eexpecto/bin/Release/eexpecto.exe --run  aa --summary                                                                                 
...
Passed:  3
	tl1/tc1
	tl1/tc2
	tc3
```

This is nuget 4.0.0

**Comment by AnthonyLloyd** [comment]
Hi @fbehrens I can't reproduce this bug. I'm looking into it now.

**Comment by AnthonyLloyd** [comment]
Hi @fbehrens, I've put in a PR fix for this.

No real workaround if you need to filter at the command line (you can focus in code though). You'll need to go back a version or wait for the release. Should be soon, my guess is within the next 24hs.

**Comment by fbehrens** [comment]
Thanks.  Fine with me,  no need to hurry. Just continue the good work.
Working '--version'  flag would be nice.  It shows in the '--help'  but
doesn't work.

Anthony Lloyd <notifications@github.com> schrieb am Sa., 18. Feb. 2017,
18:23:

**Comment by haf** [comment]
Released v4.0.1
